aip/cloud/new.md

```diff
@@ -0,0 +1,123 @@
+---
+aip:
+  id: 2511
+  scope: cloud
+  state: draft
+  created: 2019-11-22
+permlink: /cloud/2511
+redirect_from:
+  - /2511
+---
+
+<!--
+TODO: discuss policy lifecycle? binding lifecycle?
+TODO: discuss policy conditions.
+-->
+
+# Policy API Guidance
+
+A policy is a document-based resource that, when set on a resource, controls
+some behavior for that resource and its descendants. Some examples of
+controllable behaviors are:  who can access a resource, what configurations are
+allowed for a resource and what billing account to charge for a resource’s
+usage.
+
+There are 2 types of policies, _dedicated_ and _shared_.
+
+A _dedicated_ policy shares an identifier with the resource it is bound to. IAM
+policies in GCP are an example of dedicated polices. Users "set" the policy on
+directly on the resource they intend it to apply to, and all children of the
+resource inherit the policy.
+
+A _shared_ policy is a first-class resource (with its own identifier) and can
+even have policies applied to it (like an authorization policy). Unlike other
+resources, a shared policy can be _bound_ to one or more resources, and applies
+to those resources and any descendents. An example of a shared policy is
+billing accounts which are associated with resouces that bill to them.
+
+## Standard Methods
+
+For most policy types, it makes sense to incorporate the policy name in to the
```

Comment:
Reword this sentence to use the word **should**. (e.g. "APIs **should** incorporate the policy name into the method name for most policy types.")